signIn/signIn.js

```diff
@@ -0,0 +1,107 @@
+class Model {
+    constructor() {
+        this.signUp = JSON.parse(localStorage.getItem("signUp")) || [];
+      
+    }
+
+    addUser(email, password) {
+        const user = {
+            email: email,
+            password: password
+
+        }
+
+        this.signUp.push(user);
+        this.saveToStorage(this.signUp);
+    }
+
+
+
+}
+
+class View {
+    constructor() {
+        this.form = this.getElement('.form-signup');
+        this.email = this.getElement('.email');
+        this.password = this.getElement('.password');
+        this.eyeIcon = this.getElement('.eye');
+        
+        this.localListener();
+    }
+
+    localListener() {
+        this.eyeIcon.addEventListener('click', (event) => {
+
+            event.preventDefault();
+            if (this.password.getAttribute('type') === 'password') {
+                event.target.src = "../images/icon-eye-blocked.svg";
+                this.password.setAttribute('type', 'text');
+                return;
+            }
+            event.target.src = "../images/icon-eye.svg";
+            this.password.setAttribute('type', 'password');
+        })     
+    }
+
+    getElement(className) {
+        const element = document.querySelector(className);
```

Comment:
can simply return instead of let/const. return document.querySelector(className);